core/tools/blueos_startup_update/blueos_startup_update

```diff
@@ -253,7 +270,6 @@ def update_cgroups() -> bool:
     logger.info("Running cgroup update..")
 
     # Retrieve the cmdline file
-    cmdline_file = '/boot/cmdline.txt'
     cmdline_content = load_file(cmdline_file).replace('\n','').split(' ')
```

Comment:
We should skip this commands if cmdline_field is none

Reply:
I updated the tests using cmdline/config so they emit a warning and skip